scripts/remove_edge_associations.py

```diff
@@ -0,0 +1,85 @@
+import boto3
+from botocore.exceptions import ClientError
+import time
+
+def log(message):
+    print(message, flush=True)
+
+def detach_lambda_edge_associations(distribution_id):
```

Comment:
Could we add in expected types in here to improve readability of the script? There's a few missing